src/text-buffer.coffee

```diff
@@ -18,8 +19,11 @@ class TextBuffer
   getLinesLayer: ->
     @linesLayer ?= new Layer(@getCharactersLayer(), new LinesTransform)
 
-  buildTabsLayer: (tabLength) ->
-    new Layer(@getLinesLayer(), new TabsTransform(tabLength))
+  buildHardTabsLayer: (tabLength) ->
+    new Layer(@getLinesLayer(), new HardTabsTransform(tabLength))
```

Comment:
Hard tabs should be built on top of soft-tabs I think, or vice-versa. I see what you're saying about their role being somewhat redundant.